public/data/ujjwal450.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Ujjwal Kamilya",
+  "type": "personal",
+  "bio": "Currently pursuing my masters degree in computer science.Tech and Open Source Enthusiast!",
+  "avatar": "https://github.com/ujjwal450.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ujjwal450",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ujjw4lkamilya",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ujjwal-kamilya/",
+      "icon": "linkedin"
+    }
+  ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```